frontend/src/libs/components/text-editor/libs/extensions/upperline/upperline.extension.ts

```diff
@@ -0,0 +1,90 @@
+import { type CommandProps } from '@tiptap/react';
+import { Mark, mergeAttributes } from '@tiptap/react';
+
+import { UPPERLINE_EXTENSION_NAME } from './libs/constants/constants.js';
+import { type UpperlineOptions } from './libs/types/types.js';
+
+declare module '@tiptap/core' {
+  interface Commands<ReturnType> {
+    upperline: {
+      /**
+       * Set an upperline mark
+       */
+      setUpperline: () => ReturnType;
+      /**
+       * Toggle an upperline mark
+       */
+      toggleUpperline: () => ReturnType;
+      /**
+       * Unset an upperline mark
+       */
+      unsetUpperline: () => ReturnType;
```

Comment:
I think, these comments are redundant because they tell right what is written in methods names